easypost/easypost_client.py

```diff
@@ -22,7 +30,15 @@ def __init__(self, api_key: str, api_base: str = API_BASE, timeout: int = TIMEOU
         # Services
         self.address = AddressService(self)
         self.batch = BatchService(self)
+        self.beta_carrier_metadata = BetaCarrierMetadataService(self)
```

Comment:
Thought: Since users are only accessing this instance of the Carrier Metadata service/function group via the `beta_carrier_metadata` property, we don't have to call the class `BetaCarrierMetadata` since users only know it as `.beta_carrier_metadata`

Reply:
Until it goes GA, then we'd need both namespaces since for a time they will both exist together and logic could differ (which they will, the retrieve function name will change)